m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2643,10 +2671,27 @@ END [PROCEDURE]
 
 PROCEDURE BNOPRINTDEBUG
 [* Add debug messages to JSON output
-**  PASS    DEBUGMODE  Boolean - is debug mode on
+**  If the "DB:" parameter is set to false, then force Debug mode
+**    setting to off (overrides all other Debug mode logic settings).
+**  If the "DB:" parameter is set to true or the installation date was found and
+**    less than [AUTODEBUGDAYS] days prior, set the Debug mode setting to true.
+**  PASS    DEBUGMODE  Boolean
 **  PASS    BNODEBUGCOUNT  Number of debut lines to send to JSON output
 **  RETURN  BNODEBUGMSGLIST  debug output
 *]
+ IF LASTMODDATE+AUTODEBUGDAYS>SYSTEMDATE THEN
+  DEBUGMODE=DEBUGMODEON
+
+ IF PARAMDEBUGMODE=FALSE THEN
+  DEBUGMODE=FALSE
```

Comment:
I think you should be consistent with using DEBUGMODEON/DEBUGMODEOFF and not TRUE/FALSE.